widget_driver/example/lib/widgets/playground/playground_page.dart

```diff
@@ -0,0 +1,167 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_driver/widget_driver.dart';
+
+class PlaygroundPage extends StatelessWidget {
+  const PlaygroundPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ColorWidget();
+  }
+}
+
+// ########### ColorWidget Driver stuff ###########
```

Comment:
And we should remember not to commit this code later.. so we need to revert this